### DISCUSSION TITLE
Proposal CSVからのスポンサーセッションの除外

### DISCUSSION THREAD

**Discussion by nabemasat** [cloudnativedaysjp/dreamkast#1528] [Ideas]
- 現状Proposalとスポンサーセッションのエントリーを見分ける方法が、[管理画面](https://event.cloudnativedays.jp/cndt2022/admin/proposals)からの目視しかない
- CFP実施期間とスポンサーセッションの入力期間が離れていれば支障ないが、近接していると両者が混在しPromotionチームの作業自動化の妨げとなる
- そのため、出力CSVにスポンサー列を作るか、スポンサーセッションを除外したリスト出力ができると望ましい
- （鍋島の私見）スポンサーセッションの情報も出力できた方が汎用性が高いので、CSVに列を追加する方法が良いかと思っています

**Comment by jacopen** [comment]
既に採択しちゃってますが、これは比較的簡単に実装出来るので忘れないうちにしゅっとやっちゃう方がいいかも

**Comment by takaishi** [comment]
やっときますね。 https://github.com/cloudnativedaysjp/dreamkast/issues/1539 で進めます。
